doc/DOCUMENTATION.org

```diff
@@ -421,7 +421,7 @@ contribution guidelines in [[./CONTRIBUTE.org][CONTRIBUTE.org]].
 
 ** Example: Themes Megapack example
 This is a simple =contrib= configuration layer listing a bunch of themes, you
```

Comment:
Well, I only wanted to fix the wrong references and not change (core) terminology :-)